packages/graphql-language-service-server/src/MessageProcessor.js

```diff
@@ -453,8 +459,15 @@ function convertToRpcMessage(metaMessage: Object): ResponseMessage {
   return message;
 }
 
-function invalidateCache(textDocument: Object, content: Object): void {
-  const uri = textDocument.uri;
+function invalidateCache(
+  textDocument: Object,
+  uri: Uri,
+  content: Object,
+): void {
+  if (!uri) {
```

Comment:
The `Uri` type isn't nullable (it's an alias for `string`). So is this a check to see whether `uri` is an empty string?